### PR TITLE
[RHDEVDOCS-4231] bump up versions for Pipelines 1.8 release

### PR DESCRIPTION
**Pull request by SNiemann15**
**Don't merge before Pipelines 1.8 GA planned for 8/~~10/18~~22!**
Links must be verified shortly before merging!

OCP version for cherry-picking: enterprise-4.11 and later. 

Jira: https://issues.redhat.com/browse/RHDEVDOCS-4231
Versions from[ Pipeline Component Matrix](https://spaces.redhat.com/pages/viewpage.action?spaceKey=SRVKP&title=Pipeline+Component+Matrix)

Preview (VPN needed to access): http://file.str.redhat.com/~sniemann/pipelines18_tkn/cli_reference/tkn_cli/installing-tkn.html

QE review: Shubham

Version(s):

Issue:

Link to docs preview:

Additional information:

### CODE REVIEW

File: modules/op-installing-tkn-on-linux.adoc [hunk 1]
```diff
@@ -14,11 +14,13 @@ For Linux distributions, you can download the CLI directly as a `tar.gz` archive
 
 . Download the relevant CLI.
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.23.1/tkn-linux-amd64-0.23.1.tar.gz[Linux (x86_64, amd64)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-linux-amd64-0.24.0.tar.gz[Linux (x86_64, amd64)]
```
Comment (concaf) [inline modules/op-installing-tkn-on-linux.adoc:17]:
```suggestion
* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
```
the filename for this (and other architectures in the PR) will be different from this time on.

**Review by concaf — APPROVED** [review]

**Comment by concaf** [comment]
lgtm 🙃

**Review by rolfedh — APPROVED** [review]

**Comment by SNiemann15** [comment]
/assign @ppitonak 
Please review and approve.

**Review by vdemeester — APPROVED** [review]

**Review by VeereshAradhya — APPROVED** [review]

**Pull request merged**